### DISCUSSION TITLE
GL-C-008P not updating firmware?

### DISCUSSION THREAD

**Discussion by proffalken** [Koenkk/zigbee2mqtt#9591] [Devices]
Hey all,

I've got three Gledopto GL-C-008P RGBCCT devices and one of them won't update the firmware.

It's currently stuck on `V_1_1` dated `20201216` whereas the other, newer units are on `10976802` dated `20210714`.

The error message isn't on screen for long, however it appears to be `device did not request any image blocks`

Is there any way of debugging this further to try and understand what's going on here?

Other devices are updating fine, but I'm struggling to find any advice on the discussions or issues that might be causing this other than to upgrade the spec of the device Zigbee2mqtt is running on.  It's currently running in a Docker Container with USB Passthrough to the host and it has 1024Mhz CPU/1024M Ram allocated to it.

**Comment by BlueScreenTT** [comment]
did you fix this ?

i got the same problem with the 007 model

**Reply by proffalken** [reply]
Nope, not yet, I guess if there are other modules out there then this is one of the ones that doesn't report properly...

**Comment by malaakso** [comment]
I believe there are at least two hardware variants of the GL-C-008P (version 2 and 3), probably supporting different SW versions. I have version 2 running V_1_4 and version 3 running V_1_6H2. V_1_4 seems the latest available in zigbee2mqtt.

**Comment by BlueScreenTT** [comment]
Ahh

Den ons 12 jan. 2022 08:07Matti Laakso ***@***.***> skrev:
